test_wazuh/test_fim/test_skip/data/remove_nfs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+umount -f -l /nfs-mount-point
```

Comment:
This script should work in Centos, Debian and FreeBSD.

Reply:
Done